src/components/common/layout/tests/base-integration.test.ts

```diff
@@ -56,35 +56,65 @@ describe("Integration:: base page ", () => {
     await request(app).get(PATH_NAMES.SIGN_IN_OR_CREATE).expect(200);
   });
 
-  it("The footer should appear on the page when the channel is set to 'web'", async () => {
-    await setupApp(CHANNEL.WEB);
-    const response = await request(app).get(PATH_NAMES.SIGN_IN_OR_CREATE);
-    expect(response.status).to.equal(200);
-    const $ = cheerio.load(response.text);
-    expect($(".govuk-footer").length).to.equal(1);
-  });
+  describe("Web channel", () => {
+    it("should render the default govukHeader", async () => {
+      await setupApp(CHANNEL.WEB);
+      const response = await request(app).get(PATH_NAMES.SIGN_IN_OR_CREATE);
+      expect(response.status).to.equal(200);
+      const $ = cheerio.load(response.text);
+      expect($("a.govuk-header__link").length).to.equal(1);
+    });
+
+    it("should render the beta banner", async () => {
+      await setupApp(CHANNEL.WEB);
+      const response = await request(app).get(PATH_NAMES.SIGN_IN_OR_CREATE);
+      expect(response.status).to.equal(200);
+      const $ = cheerio.load(response.text);
+      expect($(".govuk-phase-banner").length).to.equal(1);
+    });
 
-  it("The footer should not appear on the page when the channel is set to 'strategic_app'", async () => {
-    await setupApp(CHANNEL.STRATEGIC_APP);
-    const response = await request(app).get(PATH_NAMES.SIGN_IN_OR_CREATE);
-    expect(response.status).to.equal(200);
-    const $ = cheerio.load(response.text);
-    expect($(".govuk-footer").length).to.equal(0);
+    it("should render the footer", async () => {
+      await setupApp(CHANNEL.WEB);
+      const response = await request(app).get(PATH_NAMES.SIGN_IN_OR_CREATE);
+      expect(response.status).to.equal(200);
+      const $ = cheerio.load(response.text);
+      expect($(".govuk-footer").length).to.equal(1);
+    });
   });
 
-  it("The beta banner should appear on the page when the channel is set to 'web'", async () => {
-    await setupApp(CHANNEL.WEB);
-    const response = await request(app).get(PATH_NAMES.SIGN_IN_OR_CREATE);
-    expect(response.status).to.equal(200);
-    const $ = cheerio.load(response.text);
-    expect($(".govuk-phase-banner").length).to.equal(1);
+  describe("Strategic App channel", () => {
+    it("should render the custom header with no links", async () => {
+      await setupApp(CHANNEL.STRATEGIC_APP);
+      const response = await request(app).get(PATH_NAMES.SIGN_IN_OR_CREATE);
+      expect(response.status).to.equal(200);
+      const $ = cheerio.load(response.text);
+      expect($("a.govuk-header__link").length).to.equal(0);
```

Comment:
Hi @di-aholme, I'm wondering if it might be worth introducing an additional `expect()` within this `it()` to check for the presence of the new header. It's great that there's an `expect` checking that the link isn't there, but I'm thinking it would be good to also check that the header _is_ there.

I'm thinking of something along the lines of 

```javascript
expect($(".strategic-app-header").length).to.equal(1);
```